Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp

```diff
@@ -236,34 +252,34 @@ SpectralKSpace::getModifiedKComponent( const DistributionMapping& dm,
 Vector<Real>
 getFonbergStencilCoefficients( const int n_order, const bool nodal )
```

Comment:
If you look at the [changes since the last review](https://github.com/ECP-WarpX/WarpX/pull/1169/files/b9c6c75ade42e2f6f9ebc1d25bbb02cf7fa125d6..135c276d85fb837445c7fc49db9e7a43590616e9#diff-be8c6fd4c00dfa144b09ef2494931aeeR252-R259), you will notice that in some of the lines of this function body two whitespaces were removed (instead of only one, as done correctly in some other lines). As a result, before we had one whitespace more for the indentation of this function body, and now we have one whitespace less, in most of the lines. The indentation rule that we try to follow in WarpX is four whitespaces in total.

The reason for insisting on this is that nothing was changed in this function body and it is easier for other reviewers to avoid confusion if we make sure that no changes appear to be highlighted here. Also, the four whitespaces indentation is one of the very few WarpX's style guidelines that is actually applied in most parts of the code (many of the other style guidelines are not), so it would be better not to break it here.

This is probably caused by the text editor doing something weird. One way to fix this, without loosing too much time, is:
1. Go through this function body and make sure that you use four whitespaces for indentation;
2. Look at the git difference between your branch and the upstream development branch with:
   ```
   git diff upstream/development Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp
   ```
3. Crosscheck that no changes are highlighted in this last part of the file, as (i) you did not modify the function implementation and (ii) we do not want to modify the indentation either (as it was correct in the first place);
4. Commit your changes and push.

PS: These kinds of comments are not a big deal (nor any kind of science, for sure) and we can of course let things go, but it is good practice to try to make the code development part of the whole effort as straightforward and clean as possible.

Reply:
Ok, I think all the whitespace should be unchanged now.